docs/release_notes/0.82.0.md

```diff
@@ -0,0 +1,14 @@
+# Release 0.82.0
+- Improve IPv6 docs and link to docs in help text (#4689)
+
+## Bug Fixes
+
+- Use correct kubeconfig for Karpenter install (#4704)
+- Fix bug causing the identityProviders to be marshalled incorrectly (#4688)
+- Fix EC2 instance selector (#4681)
+- Add ec2:DescribeVpcPeeringConnections node policy for albIngress (#4673)
+
```

Comment:
is this all? I thought we did more 🤔 , please check since last time we missed Gergely's karpenter tickets from release notes

Reply:
seems correct to me looking at the git history

Reply:
we have a lot of open PRs atm that haven't made it in

Reply:
I merged my `Avoid force-pushing to docs branch to publish docs` PR right after this PR was opened but we don't need to include it as it's not user-facing.